.golangci.yml

```diff
@@ -51,10 +45,12 @@ linters:
     - nlreturn          # generous whitespace violates house style
     - nonamedreturns    # named returns are fine; it's *bare* returns that are bad
     - protogetter       # too many false positives
+    - tenv              # replaced by usetesting
     - testpackage       # internal tests are fine
     - wrapcheck         # don't _always_ need to wrap errors
     - wsl               # generous whitespace violates house style
 issues:
+  exclude-dirs-use-default: false
```

Comment:
Whats this?

Reply:
A long time the old option was deprecated - if you run a newer linter on the old config you can see the warning.